kolena/classification/multiclass/_utils.py

```diff
@@ -55,13 +55,13 @@ def roc_curve(y_true: List[int], y_score: List[float]) -> Tuple[List[float], Lis
 
     # Map cumulative sums to tpr and fpr
     if fps[-1] <= 0:
-        log.warn("No negative samples in y_true, false positive value should be meaningless")
+        # No negative samples in y_true, false positive value should be meaningless
```

Comment:
Also note that I'm dropping these warning messages as they were spamming the log.